presto-native-execution/presto_cpp/main/operators/ShuffleWrite.cpp

```diff
@@ -22,6 +22,15 @@ velox::core::PlanNodeId deserializePlanNodeId(const folly::dynamic& obj) {
   return obj["id"].asString();
 }
 
+#define CALL_SHUFFLE(call, methodName)                                \
+  try {                                                               \
+    call;                                                             \
+  } catch (const VeloxException& e) {                                 \
+    throw;                                                            \
+  } catch (const std::exception& e) {                                 \
```

Comment:
Better to use catch-all `catch (...)` ?

Reply:
@vermapratyush Curious why would this be better? How would we get an error message from ...?

Reply:
@mbasmanova not replacement but in addition to std::exception.
That can be helpful if there are operators which throw non std exception (eg folly::exception_wrapper). Instead of error message we could have a stack trace (atleast).

Reply:
I'm not familiar with folly::exception_wrapper. Need to take a closer look and follow up.